src/plugins/minimal_scene/MinimalScene.cc

```diff
@@ -466,6 +466,9 @@ void IgnRenderer::Initialize()
 
   std::map<std::string, std::string> params;
   params["useCurrentGLContext"] = "1";
+  params["winID"] = std::to_string(
+    ignition::gui::App()->findChild<ignition::gui::MainWindow *>()->
+      QuickWindow()->winId());
```

Comment:
I know `Scene3D` is deprecated, but how about doing the same there? For users who are transitioning from one to the other. The same for `GzScene3D`